docs/backend/installation.md

```diff
@@ -0,0 +1,128 @@
+# Instalation
+
+<div class="Alert Alert--warning">
+
+This manual is prepared for the developer installation
+
+</div>
+
+## Requirements
+---
+
+| Name                  | Version                           |
+|-----------------------|-----------------------------------|
+| Language              | Php >= 7.2                        |
+| Database              | Posgress >= 10                    |
+| Message Broker*       | RabbitMQ >= 3.6.10                |  
+
+*Use for running application in asynchronous mode.
+
+## Quick Start
+---
+
+Clone [backend project repository][repository] to your local directory:
+```
+git clone git@github.com:ergonode/backend.git
+```
+
+<div class="Alert Alert--info">
+
+Application on default starts on **development** branch
+
+</div>
+
+If want run last stable application version execute:
+```
+git checkout master
+``` 
+In .env file you need to configure database connection
+
+
+Open your terminal in local project, and execute:
+```
+composer install
+``` 
+In .env file you need to configure database connection
+```
+DATABASE_URL=pgsql://db_user:db_password@127.0.0.1:5432/db_name
+```
+Generate keys
+```
+openssl genrsa -out config/jwt/private.pem -aes256 4096
+openssl rsa -pubout -in config/jwt/private.pem -out config/jwt/public.pem
+```
+While executing above commends you would be asked about password. This password needs to be saved then in .env file in line `JWT_PASSPHRASE=yourpassword`
+
+In terminal execute command which configure application (Available phing commands):
```

Comment:
In terminal, execute command which configures application (Available phing commands):